packages/aragon-wrapper/src/index.test.js

```diff
@@ -360,69 +360,89 @@ appInitTestCases.forEach(([testName, permissionsObj]) => {
       call: sinon.stub().withArgs('KERNEL_APP_ID').resolves('kernel'),
       events: sinon.stub().returns(of([]))
     }
+
     // act
     await instance.initApps()
+
     // assert
+    // Check value of apps
+    return new Promise(resolve => {
+      instance.apps.pipe(first()).subscribe(value => {
+        t.deepEqual(value, [
+          {
+            abi: 'abi for kernel',
+            appId: 'kernel',
+            codeAddress: '0xkernel',
+            isAragonOsInternalApp: true,
+            proxyAddress: '0x123'
+          }, {
+            abi: 'abi for counterApp',
+            appId: 'counterApp',
+            codeAddress: '0xcounterApp',
+            isForwarder: false,
+            kernelAddress: '0x123',
+            proxyAddress: '0x456'
+          }, {
+            abi: 'abi for votingApp',
+            appId: 'votingApp',
+            codeAddress: '0xvotingApp',
+            isForwarder: false,
+            kernelAddress: '0x123',
+            proxyAddress: '0x789'
+          }
+        ])
+        resolve()
+      })
+    })
+    })
+})
 
-    // Check initial value of apps
-    instance.apps.pipe(first()).subscribe(value => {
-      t.deepEqual(value, [
-        {
-          abi: 'abi for kernel',
-          appId: 'kernel',
-          codeAddress: '0xkernel',
-          isAragonOsInternalApp: true,
-          proxyAddress: '0x123'
-        }, {
-          abi: 'abi for counterApp',
-          appId: 'counterApp',
-          codeAddress: '0xcounterApp',
-          isForwarder: false,
-          kernelAddress: '0x123',
-          proxyAddress: '0x456'
-        }, {
-          abi: 'abi for votingApp',
-          appId: 'votingApp',
-          codeAddress: '0xvotingApp',
-          isForwarder: false,
-          kernelAddress: '0x123',
-          proxyAddress: '0x789'
-        }
-      ])
+test('should init the app identifiers correctly', async (t) => {
+  t.plan(1)
+  // arrange
+  const { Aragon } = t.context
+  const instance = new Aragon()
+  // act
+  await instance.initAppIdentifiers()
+  // assert
+  instance.identifiers.subscribe(value => {
+    t.deepEqual(value, {})
```

Comment:
What's the reason that we emit an empty object initially? Is that due to the consumers rely on there always being some value (at times an empty object like here)

Reply:
No particular reason, but an empty one seems reasonable to begin with, API-wise, since the app identifier mapping should alway "exist" (but may not have any identifiers declared inside it).